debian/control

```diff
@@ -9,7 +9,7 @@ Build-Depends: bash-completion,
                dh-systemd | debhelper (>= 13.3),
                gettext,
                git,
-               golang,
+               golang-1.10 | golang,
```

Comment:
If this works how I hope it does, this might actually fix our architecture support woes as well. Since it appears that `golang-1.10` supports all architecures on those releases where `golang` does not (aka where `golang` is a version lower than 1.10).

Reply:
@lucasmoura @blackboxsw What is the best way to test the builds for the architectures that are currently failing?

Reply:
I retract my claim that this would fix the arch problem. I was looking at the wrong thing :upside_down_face: